meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb

```diff
@@ -1,5 +1,5 @@
 SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "d0ca9d072012d8ac333537d9b922d9242e022efe"
+SRCREV = "79a4f69e7588a6efc4b4c95abec7794216588087"
```

Comment:
As I tried to explain above, from the **point of view of m-g-d** (which is what you are changing here), change of name is a side-effect. It's very similar to submodule updates in g-d-p repo and you can see from the log (and our new script) how we don't mention the actual changes in the shortlog but rather in the description.

Anyway, I'll modify the commit log for you.